<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="dataTableView">
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QChartView" name="chartView" native="true"/>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>220</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Testing</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Test Time(s):</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QSpinBox" name="testTime">
               <property name="minimum">
                <number>60</number>
               </property>
               <property name="maximum">
                <number>360000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Interval(s):</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QComboBox" name="interval">
               <item>
                <property name="text">
                 <string>0.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>30</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>60</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>100</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="startButton">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QPushButton" name="singleTestButton">
               <property name="text">
                <string>Single Test</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="newTest">
               <property name="text">
                <string>New Test</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="saveButton">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>GroupBox</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="0">
              <widget class="QPushButton" name="set100T">
               <property name="text">
                <string>100%T</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="set0T">
               <property name="text">
                <string>0%T</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="mode">
               <item>
                <property name="text">
                 <string>Average</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="0" colspan="3">
              <widget class="QLCDNumber" name="lcdNumber">
               <property name="mode">
                <enum>QLCDNumber::Dec</enum>
               </property>
               <property name="segmentStyle">
                <enum>QLCDNumber::Flat</enum>
               </property>
               <property name="value" stdset="0">
                <double>0.001000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>lcdNumber</zorder>
            <zorder>set100T</zorder>
            <zorder>set0T</zorder>
            <zorder>mode</zorder>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Boards</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>2</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QToolButton" name="BoardOneUp">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/up_arrow.png</normaloff>:/up_arrow.png</iconset>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QToolButton" name="BoardTwoUp">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/up_arrow.png</normaloff>:/up_arrow.png</iconset>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QToolButton" name="BoardOneStop">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/IMG_2183.PNG</normaloff>:/IMG_2183.PNG</iconset>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QToolButton" name="BoardTwoStop">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/IMG_2183.PNG</normaloff>:/IMG_2183.PNG</iconset>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QToolButton" name="BoardOneDown">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/down_arrow.png</normaloff>:/down_arrow.png</iconset>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QToolButton" name="BoardTwoDown">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/down_arrow.png</normaloff>:/down_arrow.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Settings</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Data path:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="pathLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseButton">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSetting"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
